app/presenters/hyrax/manifest_components.rb

```diff
@@ -0,0 +1,78 @@
+module Hyrax
+  module ManifestComponents
```

Comment:
Rather than add these methods to the presenter via a module, could we extract a Manifest class that the presenter has an instance of?

Reply:
`iiif_manifest` calls various methods directory on the object it receives, which is the presenter hence the methods being added to presenter?

Reply:
That's fine. Can we pass a reference to the presenter?

Reply:
Alternatively, can most of these methods go into the ManifestHelper class?

Reply:
Yes. ManifestComponents is gone, a couple of methods have gone back into WorkShowPresenter and the rest into ManifestHelper ... I think this is better.